### ISSUE TITLE
Add different icons for Maps and Tilesets

### ISSUE THREAD

**Issue by carreraSilvio** [mapeditor/tiled#3280]
![image](https://user-images.githubusercontent.com/3154359/154856346-49d4ad94-abf5-4ba6-95b0-5014caeab395.png)

Add an icon to differentiate a map from a tileset and hide the extensions. I think this would make working with the tool slightly easier. Another solution my be that have two parent tabs MAPS and TILESETS and just switch between them.

**Comment by eishiya** [comment]
I'd rather not separate tiles and tilesets into groups since if I have a tileset open in Tiled, it's probably because I want it near some maps. I think icons are a great idea, perhaps with the option to disable them for people who want every bit of tab real estate they can get. I'd probably keep them enabled as long as they're not too distracting :D

Maybe a "T" made out of a 3x3 tiles like the Tiled logo for tilesets, and an M in different colours on the same grid, something like these but hopefully polished by someone who knows how to do icons:
![tiledicon_tileset](https://user-images.githubusercontent.com/70545646/154999215-70a7294b-0805-4e11-8a37-d004a933b54f.png) ![tiledicon_tmap](https://user-images.githubusercontent.com/70545646/154999219-2e6dfcc2-a755-48dd-8244-22414d1edabd.png)
Using T and M is of course very Anglocentric, but I hope that the colours and shapes are distinct enough to be easy to get used to for those using Tiled in other languages.

**Comment by bjorn** [comment]
I'm not a fan of the T and M, mostly because they're so language-specific. I like the ones used by [tIDE](https://github.com/colinvella/tIDE), since they communicate some meaning: ![](https://raw.githubusercontent.com/colinvella/tIDE/master/TileMapEditor/tIDE/Resources/Map.png) ![](https://raw.githubusercontent.com/colinvella/tIDE/master/TileMapEditor/tIDE/Resources/TileSheet.png)

Probably vectorized versions based on those would do well.

**Comment by eishiya** [comment]
Agreed, symbols are much better. Not so colourful though, those would be distracting.
